_artifacts/document-attributes.adoc

```diff
@@ -91,8 +91,8 @@ endif::OP[]
 :AMQ_BROKER_VERSION: 7.8
 :AMQ_BROKER_VERSION_LONG: 7.8.1
 :CEKIT: CEKit
-:CEKIT_VERSION: 3.8
-:CEKIT_VERSION_LONG: 3.8.0
+:CEKIT_VERSION: 3.11
+:CEKIT_VERSION_LONG: 3.11.0
```

Comment:
CeKit 3.12 is released already, we can use it here.

Reply:
Thanks, updated to 3.12

Reply:
Just checked https://docs.cekit.io/en/{CEKIT_VERSION_LONG}/handbook/installation/index.html and the latest published version is 3.11.0. Do you know when 3.12.0 will be released?

Reply:
it is released already, 3.12 points to latest:
https://docs.cekit.io/en/latest/#

Reply:
@spolti Thanks, my mistake. I checked by using versions and I see now that `latest` is used instead of the version number in the URL.

@emmurphy1 What do you suggest? Do I substitute `latest` in the URL instead of the version attribute?